Library/Homebrew/extend/os/mac/pkgconf.rb

```diff
@@ -11,7 +11,7 @@ def macos_sdk_mismatch
       return if OS::Mac.version.prerelease? || OS::Mac.version.outdated_release?
 
       pkgconf = begin
-        ::Formula["pkgconf"]
+        ::Formulary.factory("pkgconf", prefer_stub: true)
```

Comment:
```suggestion
        ::Formulary.stubbary("pkgconf")
```
@Rylan12 almost wonder if a helper method like this would be nice?